.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this intended to be for JRE and Android? Don't we need a README that explains how to build for each environment?

I'm not sure it really helps to have this as a Studio project, and I don't think we should require people to install it just to try out the tutorial - maybe a minimal setup with a `build.gradle` would be simplest, and people can import it into Studio if that's their preferred environment.